### ISSUE TITLE
approvable

### ISSUE THREAD

**Issue by geetarista** [plataformatec/devise#223]
I'm wondering what everyone thinks about the ability to have an :approvable module. For sites that don't want to be public facing, it would be nice to use the :registerable module, but make it so accounts have to be approved before activation.

Not sure if this is outside the scope of devise, but I think it would be nice to have this ability or something similar.
